### DISCUSSION TITLE
Multi-line mathematics doesn't work correctly

### DISCUSSION THREAD

**Discussion by ellafm** [mermaid-js/mermaid#5885] [Q&A]
The first set of examples given in the help guide here are not rendered correctly (tested in Chrome, Edge)
[https://mermaid.js.org/config/math.html](https://mermaid.js.org/config/math.html)

````{md}
```{mermaid}
 graph LR
      A["$$x^2$$"] -->|"$$\sqrt{x+3}$$"| B("$$\frac{1}{2}$$")
      A -->|"$$\overbrace{a+b+c}^{\text{note}}$$"| C("$$\pi r^2$$")
      B --> D("$$x = \begin{cases} a &\text{if } b \\ c &\text{if } d \end{cases}$$")
      C --> E("$$x(t)=c_1\begin{bmatrix}-\cos{t}+\sin{t}\\ 2\cos{t} \end{bmatrix}e^{2t}$$")

```
````
![snapshot1](https://github.com/user-attachments/assets/0fefda73-047b-4f1c-bcfb-c345959a01d7)

Line breaks using `<br/>` also do not work when the text includes math expressions such as $$x^2$$

````{md}
```{mermaid}
flowchart LR
  A["$$x^2$$"] --> B("$$x^2$$ <br/> Hello")
  B --> C{Decision}
  C --> D[Result one]
  C --> E[Result two]
```
````
![mermaid-diagram-2024-09-24-092149](https://github.com/user-attachments/assets/cf3724b8-1f5a-41c3-9967-4cee49a05a8d)

**Comment by ellafm** [comment]
I've discovered that the first example can be addressed by replacing `\\` with `\\\` in maths expressions requiring newline characters.

**Comment by swalter972** [comment]
Any hints ? I would need this for my student diagrams. Thanks ( KteX support does not help

\\, \cr, \\[distance], and \cr[distance]. Distance can be written with any of the [KaTeX units](https://katex.org/docs/supported.html#units).
